libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/EntityAssignmentComparer.cs

```diff
@@ -31,6 +31,12 @@ public EntityAssignmentComparer(string[] operations)
             operationPreference = operations;
         }
 
+        /// <summary>
+        /// Compares <see cref="EntityAssignment"/> x against y to determine its relative priority.
+        /// </summary>
+        /// <param name="x">Reference Entity.</param>
+        /// <param name="y">Comparisson Entity.</param>
+        /// <returns>Numerical value representing x's relative priority.</returns>
```

Comment:
Same here, maybe `reference's relative...` instead of `x's relative...`?

Reply:
Ah, gotcha, the remark is related to x and y being used as generic references. I used x and y since those were the parameters, I still think its more helpful this way.